.gitignore

```diff
@@ -16,3 +16,8 @@ data
 /.idea/
 *.ipr
 *.iml
+
+*.crt
+*.key
+*.pem
+*.json
```

Comment:
I feel like blacklisting every json file could backfire

Reply:
Do not think, we need json inside the repo. We can enable or `--focre` them anyway.

Reply:
I disagree. It will be extremely hard for potential future contributors that have not seen this PR to know why `git add .` misses their json file.